.github/workflows/ci.yaml

```
@@ -0,0 +1,68 @@
+name: "CI"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master 
+  create:
+    tags:
+      - v*
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  cancel:
+    name: "Cancel"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Get workflow id"
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+  build:
+    name: "Build"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    steps:
+
+      - name: Set up cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            $HOME/.ghc
+            $HOME/.cabal
+            $HOME/.stack
+          key: ${{ runner.os }}-haskell
+
+      - name: Set up environment
+        uses: actions/setup-haskell@v1.1
+        with:
+          ghc-version: 'latest'
+          enable-stack: true
+          stack-version: 'latest'
+
+      - name: Verify environment
+        run: |
+          stack --numeric-version
+          ghc --version
+          
+      - name: Build dependencies 
+        run: |
+          cd waspc
+          stack test --only-dependencies
```

```suggestion
          stack --install-ghc test --only-dependencies
```
Based on https://github.com/wasp-lang/wasp/pull/130/files/bfb096581371171c14179d231d1558c18120683f#r521635654 .